drivers/stepper/Kconfig.gpio

```diff
@@ -0,0 +1,13 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 Carl Zeiss Meditec AG
+# SPDX-License-Identifier: Apache-2.0
+
+menu "GPIO STEPPER Driver"
+
+config GPIO_STEPPER
+	bool "Activate driver for GPIO_STEPPER_MOTOR_CONTROLLER"
+	depends on DT_HAS_GPIO_STEPPER_ENABLED
+	default y
+	help
+	  GPIO_STEPPER_MOTOR_CONTROLLER driver for stepper motor control
```

Comment:
that's a weird description and help text, just "GPIO stepper controller", though it would be nice to make it clear that this is NOT for a step-direction type of controller, maybe "Dual H-bridge stepper motor controller"? And I guess there will be a "Step-direction stepper motor controller" down the road.